<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SelfTech-027" ignored="1" name="Error Validations Test" tests="2" failures="1" timestamp="2023-01-09T23:17:11 IST" time="0.116" errors="0">
  <testcase name="LoginErrorValidation" time="0.019" classname="RahulShetty.SeleniumFrameworkShopping.ErrorValidations">
    <skipped/>
  </testcase> <!-- LoginErrorValidation -->
  <testcase name="LoginErrorValidation" time="0.003" classname="RahulShetty.SeleniumFrameworkShopping.ErrorValidations">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at RahulShetty.SeleniumFrameworkShopping.ErrorValidations.LoginErrorValidation(ErrorValidations.java:21)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- LoginErrorValidation -->
  <testcase name="ProductErrorVaidation" time="0.0" classname="RahulShetty.SeleniumFrameworkShopping.ErrorValidations">
    <ignored/>
  </testcase> <!-- ProductErrorVaidation -->
</testsuite> <!-- Error Validations Test -->
